.gitattributes

```diff
@@ -13,3 +13,4 @@
 *.ttf       binary
 *.ogg       binary
 *.wav       binary
+*.icns	    binary
```

Comment:
Is this an alternative extension to .ico?

Reply:
Nope, this is a specific "Icon set" format for OSX/iOS (providing different copies of the icon at different res)